src/Swarm/Language/Parse.hs

```diff
@@ -242,6 +243,10 @@ parseTermAtom =
         <|> TInt <$> integer
         <|> TString <$> stringLiteral
         <|> TBool <$> ((True <$ reserved "true") <|> (False <$ reserved "false"))
+        <|> reserved "require"
+          *> ( (TRequireDevice <$> stringLiteral)
+                <|> (TRequire <$> (fromIntegral <$> integer) <*> stringLiteral)
+             )
```

Comment:
Could you add a help text to the `stringLiteral` parser?

It is not easy to understand the error message when I write `\x. require 1 x` and get:
```
Error: unexpected 'x', expecting '"'
```
Something like expecting `"a literal string that is the name of an entity"` or specifically "device" in the first case, would be more helpful IMO. :slightly_smiling_face:

Reply:
Good idea, thanks! :heavy_check_mark: